sqldelight-compiler/integration-tests/src/test/kotlin/com/example/PlayerQueries.kt

```diff
@@ -80,7 +79,7 @@ class PlayerQueries(private val queryWrapper: QueryWrapper, private val database
             Player::Impl)
 
     fun <T : Any> selectNull(mapper: (expr: Void?) -> T): Query<T> = Query(85, selectNull, database,
-            "SELECT NULL") { cursor ->
+            { "SELECT NULL" }) { cursor ->
```

Comment:
Should there be an overload for when the string is a literal? Turning all of these into anonymous classes when they don't really need to be is going to have an adverse impact on method count.

Reply:
im cool with that, or just doing the string literal always if it's not actually that much of an efficiency gain. Thoughts?